dojo/tools/hcl_asoc_sast/parser.py

```diff
@@ -0,0 +1,156 @@
+from xml.dom import NamespaceErr
+
+from defusedxml import ElementTree as ET
+
+from dojo.models import Finding
+
+
+class HCLASoCSASTParser:
+    def get_scan_types(self):
+        return ["HCL AppScan on Cloud SAST XML"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Import XML output of HCL AppScan on Cloud SAST"
+
+    def xmltreehelper(self, input):
+        if input.text is None:
+            output = None
+        elif "\n" in input.text:
+            output = ""
+            for i in input:
+                output = output + " " + i.text
+        else:
+            output = " " + input.text
+        return output
+
+    def get_findings(self, file, test):
+        findings = []
+        tree = ET.parse(file)
+        root = tree.getroot()
+        if "xml-report" not in root.tag:
+            msg = "This doesn't seem to be a valid HCL ASoC SAST xml file."
+            raise NamespaceErr(msg)
+        report = root.find("issue-group")
+        if report is not None:
```

Comment:
Instead of putting the whole function after this point inside an `if` block when `report` is *not* `None`, just bail if `report` *is* `None`.
```suggestion
        if report is None:
            return findings
```

Reply:
I was keeping the overall style of the parser similar to the existing hcl_appscan (for DAST) parser for consistency.  I can refactor if this is a dealbreaker.